programs/system/src/instructions/create_account.rs

```diff
@@ -28,7 +30,26 @@ pub struct CreateAccount<'a> {
     pub owner: &'a Pubkey,
 }
 
-impl CreateAccount<'_> {
+impl<'a> CreateAccount<'a> {
+    pub fn with_rent_check(
```

Comment:
What do you think if we rename this to:
```suggestion
    pub fn with_minimal_balance(
```
This might reflect better what the method does, since it is not technically doing a rent "check".